src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1209,7 +1209,7 @@ private void updateOverallHappiness()
             final BuildingHome home = citizen.getHomeBuilding();
             if(home != null)
             {
-                housing = home.getBuildingLevel();
+                housing += home.getBuildingLevel();
```

Comment:
@WORMSS aggreed in some case, some house would not be counted.
@Raycoms I believe you wrote this, could you comment ?